requirements.in

```diff
@@ -19,7 +19,7 @@ django-tinymce==3.4.0
 djangorestframework==3.12.2
 drf-oidc-auth==0.10.0
 drf-spectacular==0.24.2
-easy-thumbnails==2.7.1
+easy-thumbnails==2.8.3
```

Comment:
I think we can we unpin the version while we for this?

Reply:
Removed 👍